### ISSUE TITLE
Can't send any transactions

### ISSUE THREAD

**Issue by jabo38** [NewEconomyMovement/NEMiOSApp#3]
Each time I try to send a transaction, the app crashes.

**Comment by jabo38** [comment]
Update: after trying and trying to send some transactions, I finally was able to do some sending.  

The only time I was able to get an account to work was by scanning a very old QR that I had made from a version long ago.  

Anytime I create a new account (and I have tried soooooo many times), it can not make transactions.   And every time I make a new account on my desktop NCC, fund it, and load the private key on the mobile app, that account too does not work.  

So basically, other than me, who is the only one that has a few QRs from a long time ago, nobody else can test transactions, and even I can only do it on some accounts.

**Comment by jabo38** [comment]
I have confirmed with another tester.  Any new account he makes doesn't send, but if he makes an account using one of my old QRs, then the account works.

**Comment by LyubomirDominik** [comment]
Fixed in Version 2 Build 5

**Comment by jabo38** [comment]
fixed.  transactions sending well in latest build
